recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - pip
```

Comment:
FYI version 2.3.1 is out: https://github.com/pdm-project/pdm/releases/tag/2.3.1

Reply:
Are you ok leaving this at 2.2.1 for now? Updating to 2.3.1 will require building more dependencies.